Hymns/SearchBarView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchBarView.swift
+//  Hymns
+//
+//  Created by Johanna Findeisen on 3/25/20.
+//  Copyright © 2020 skywalkerdude. All rights reserved.
+//
+
```

Comment:
Do we generally want to have these header files? In the past I haven't included them

Reply:
Good point, yeah let's do away with them.